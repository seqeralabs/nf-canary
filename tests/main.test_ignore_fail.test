nextflow_process {

    name "Test Process TEST_IGNORED_FAIL"
    script "main.nf"
    process "TEST_IGNORED_FAIL"

    test("Should run without failures") {

        then {
            assert process.success
            assert snapshot(process).match()
            assert process.trace.failed().size() == 1
            assert process.trace.succeeded().size() == 0
        }

    }

}
