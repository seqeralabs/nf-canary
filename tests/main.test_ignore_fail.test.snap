{
    "Should run without failures": {
        "content": [
            {
                "stderr": [
                    
                ],
                "errorReport": null,
                "exitStatus": 0,
                "failed": false,
                "stdout": [
                    
                ],
                "errorMessage": null,
                "trace": {
                    "tasksFailed": 1,
                    "tasksCount": 1,
                    "tasksSucceeded": 0
                },
                "name": "TEST_IGNORED_FAIL",
                "success": true
            }
        ],
        "timestamp": "2023-10-04T12:11:38.820379"
    }
}